ansible/docker.yml

```diff
@@ -18,7 +18,9 @@
       shell: curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
 
     - name: Add docker repository
-      shell: add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
```

Comment:
is this change needed? the old version is simpler to read

Reply:
it was not completing when on the older way. so it essentially  no completing the playbook.

Reply:
try `shell: add-apt-repository -y "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"`

Reply:
moved it back with "-y" parameter